packages/ete-tests/src/__test__/generate/fixtures/fern.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "workspaces": ["**"],
+  "org": "fern"
```

Comment:
cool that this leaves room for future configurations:

- authorization
- billing
- analytics